toolkit/tools/imagecustomizerapi/corruptionoption.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type CorruptionOption string
+
+const (
+	CorruptionOptionIoError CorruptionOption = "io-error"
```

Comment:
Add `CorruptionOptionDefault CorruptionOption = “”` here. And then remove the pointer from the `CorruptionOption` property in the `Verity` struct.

Reply:
Unmarshal the yaml with zero of type to look for the default arg prepared. Ok, got it. Will revise.